README.md

```diff
@@ -38,6 +38,16 @@ git clone https://github.com/$YOUR_GITHUB_USERNAME/autoscaler.git
 cd autoscaler
 ```
 
+## Building the Cluster autoscaler
+
+```shell
+# Replace "$YOUR_GITHUB_USERNAME" below with your github username
+git clone https://github.com/$YOUR_GITHUB_USERNAME/autoscaler.git
+cd autoscaler
+docker build -t autoscaler cluster-autoscaler/Dockerfile .
```

Comment:
Please check Makefile for the commands to build binary and container image.

I would suggest to use existing commands to rewrite this section. Dockerfile doesn't have to changed.

Reply:
Sure , I will probably change the Readme file and keep the Dockerfile same so that those  who are using it already do not have to change .

Reply:
@Jeffwan I have updated only the readme now, can you check ? LMK if any additions or deletions required